.pylintrc

```diff
@@ -7,7 +7,7 @@ extension-pkg-whitelist=
 
 # Add files or directories to the blacklist. They should be base names, not
 # paths.
-ignore=CVS
+ignore=CVS, 0001_initial.py, test_group_method.py, group_graph.py, 0002_auto_20190313_1424.py
```

Comment:
Is it supposed to be CVS or should it be CSV?

Reply:
CVS, that's how it came.